src/WebJobs.Script/Config/ScriptHostOptionsSetup.cs

```diff
@@ -24,6 +28,7 @@ public ScriptHostOptionsSetup(IConfiguration configuration, IEnvironment environ
             _configuration = configuration;
             _environment = environment;
             _applicationHostOptions = applicationHostOptions;
+            _languageWorkerService = new LanguageWorkerService(configuration);
```

Comment:
Rather than newing up the service here and assigning configs to ScriptOptions, you should add it as a service to ScriptHostBuilderExtensions, then it will be available everywhere:

```csharp
services.AddSingleton<ILanguageWorkerService, LanguageWorkerService>();
```

Then you don't have to stuff these configs on ScriptJobHostOptions where they don't belong.

Reply:
Removed _languageWorkerService reference from scriptHostOptions